src/libraries/Native/Unix/System.Globalization.Native/pal_icushim_internal.h

```diff
@@ -12,28 +12,34 @@
 
 #include "config.h"
 
-#define U_DISABLE_RENAMING 1
+#if defined(TARGET_ANDROID)
+    #include "pal_icushim_internal_android.h"
```

Comment:
Nit: Do we use indentation in other places for #includes and #defines?

Reply:
looks like we don't, removed.